wiki/Community/Project_Loved/en.md

```diff
@@ -17,7 +17,9 @@ To submit a beatmap for Loved, please visit the [beatmap submission page](https:
 
 Nominated beatmaps are then posted to the [Project Loved forum](https://osu.ppy.sh/community/forums/120), where community members may vote whether they want the beatmap to be Loved or not. Maps reaching 80% "Yes" votes at their polls' end time will be moved to the Loved category.
 
-Creators of said beatmaps are also able to request them to be removed from the Loved category by reaching out to a [Project Loved Team](https://osu.ppy.sh/groups/31) member or by posting in `#osu-loved` channel of the [osu! Discord server](https://discord.gg/ppy). The removal process will take a minimum of 1 week and creator will receive a follow-up when it is finished.
+Creators of said maps **cannot** request them to be removed from the Loved category. Moreover, Loved difficulties cannot directly be reused in Ranked beatmaps.
```

Comment:
here you are saying maps cannot directly be reused, but in the ranking criteria page you're saying its discouraged?

this can open a can of worms, so needs to be straightened out. i'd also suggest adding some text about handling remapped loved difficulties on a case-by-case basis (by the NAT preferably as thats their domain)

Reply:
The keyword here (and the Ranking Criteria) is **directly**. It implies you can't just `Ctrl+C`, `Ctrl+V` the exact same difficulty, and that some kind of changes are expected. This would be handled no differently that how we currently handle reusing Ranked difficulties.

I don't think we can codify the case-by-case basis you suggest. It would basically be unprecedented in the Ranking Criteria, having the NAT have the very last call on the matter. That would need to be ran through the team in their review.

Reply:
spoke to marti directly and figured it out, was a misunderstanding on my end